regression/contracts/invar_check_continue/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Again.  I prefer to check VCs as well as it catches bugs that cause VCs to be omitted.

Reply:
Good idea! I would add VC checks to all tests. Thanks!